mpa/seg/inferrer.py

```diff
@@ -95,18 +95,20 @@ def infer(self, cfg, model=None, dump_features=False):
             target_classes = dataset.CLASSES
 
         # Model
-        if model is None:
-            cfg.model.pretrained = None
-            if cfg.model.get('neck'):
-                if isinstance(cfg.model.neck, list):
-                    for neck_cfg in cfg.model.neck:
-                        if neck_cfg.get('rfp_backbone'):
-                            if neck_cfg.rfp_backbone.get('pretrained'):
-                                neck_cfg.rfp_backbone.pretrained = None
-                elif cfg.model.neck.get('rfp_backbone'):
-                    if cfg.model.neck.rfp_backbone.get('pretrained'):
-                        cfg.model.neck.rfp_backbone.pretrained = None
-            cfg.model.test_cfg.return_repr_vector = True
+        cfg.model.pretrained = None
```

Comment:
Here also.

Reply:
To be honest, I have no clue. This part is here as it is.